src/dfx/src/commands/canister/call.rs

```diff
@@ -194,9 +194,13 @@ pub fn get_effective_canister_id(
                 Decode!(arg_value, In).context("Argument is not valid for InstallChunkedCode")?;
             Ok(in_args.target_canister)
         }
-        MgmtMethod::BitcoinGetUtxosQuery | MgmtMethod::BitcoinGetBalanceQuery => {
-            Ok(CanisterId::management_canister())
-        }
+        // TODO: SDKTG-347
+        // BitcoinGetUtxosQuery and BitcoinGetBalanceQuery are to be removed.
+        // We should remove this match arm when they are removed from `MgmtMethod` in ic-utils.
+        _ => Err(anyhow!(
```

Comment:
Suggestion: leave these as explicit match patterns.

That way, once they are removed from ic-utils, we'll get a compile error here (and know to remove them).  But also, if new management methods were added, we'd also get a compile-time error.
```suggestion
        MgmtMethod::BitcoinGetUtxosQuery | MgmtMethod::BitcoinGetBalanceQuery => Err(anyhow!(
```